src/dos/drive_local.cpp

```diff
@@ -154,9 +154,15 @@ bool localDrive::FileOpen(DOS_File **file, char *name, uint32_t flags)
 		// If yes, check if the file can be opened with Read-only access:
 		fhandle = fopen_wrap(newname, "rb");
 		if (fhandle) {
+			if (!always_allow_rw_open)
+				fhandle = NULL;
```

Comment:
In >= C++11 `nullptr` instead of `NULL` (because `nullptr` is a strict type for a pointer to memory, where as `NULL` is a basic integer) https://stackoverflow.com/a/14112019

Reply:
Also - I like putting the check at this level because it still lets the debug-mode messages trigger (instead of adding the check earlier).

Reply:
Oh, right, I normally use nullptr in Staging, just forgot this time.

Reply:
np!